docs/run-testing-framework.md

```diff
@@ -271,4 +271,4 @@ cd terraform/add_on/remote_state && terraform init && terraform apply \
    -var="testing_id={{test case unique id}}" \
    -var="folder_name={{folder name when uploading to s3}} \
    -var="platform={{platform running (ec2, ecs, eks, canary,...)"\
-````
\ No newline at end of file
+````
```

Comment:
```suggestion
```
```

Reply:
Not sure what this is supposed to be - if it's to add a newline then that should be added if not already present